packages/schema/package.json

```diff
@@ -56,32 +83,38 @@
     "build:tsc": {
       "command": "tsc --pretty",
       "files": [
-        "src/",
+        "package.json",
+        "src/**/*.ts",
         "tsconfig.json",
+        "__tests__/**/*.ts",
         "../../core/tsconfig/base.json"
       ],
       "output": [
-        "dist/",
+        "dist/**",
```

Comment:
dist/** and dist/ are the same (wireit uses the same semantics as npm `files` array, so if you give it a folder, it's implicitly recursive)